src/LineageTree/lineageTree.py

```diff
@@ -1882,10 +1883,23 @@ def draw_tree_graph(
         ax.get_xaxis().set_visible(False)
         return ax
 
-    def to_simple_graph(self, node=None, start_time: int = 0):
+    def to_simple_graph(self, node=None, start_time: int = None):
+        """Generates a dictionary of graphs where the keys are the index of the graph and
+        the values are the graphs themselves which are produced by create_links_and _cycles
+
+        Args:
+            node (_type_, optional): The id of the node/nodes to produce the simple graphs. Defaults to None.
+            start_time (int, optional): Important only if there are no nodes it will produce the graph of every
+            root that starts before or at start time. Defaults to None.
+
+        Returns:
+            (dict): The keys are just index values  0-n and the values are the graphs produced.
+        """
         if node is None:
             mothers = [
-                root for root in self.roots if self.time[root] <= start_time
+                root
+                for root in self.roots
+                if self.time[root] <= start_time + self.t_b
```

Comment:
If start time is `None`, that's not gonna work (`None + int` does not work)

Reply:
You should have kept as it was but put at the beginning of the function something like:
```python
if start_time is None:
    start_time = self.t_b
```